### PR TITLE
update the 03 markdown file

### PR DESCRIPTION
**Pull request by BibhuA**
Close Issue #14

### CODE REVIEW

File: _slides/03-BibhuA.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Working with git in GitHub is awesome!
```
Comment (alanlayson) [inline _slides/03-BibhuA.md:6]: good

**Comment by alanlayson** [comment]
good

**Review by alanlayson — APPROVED** [review]
good

**Comment by BibhuA** [comment]
Done.

**Comment by BibhuA** [comment]
good job!

**Pull request closed without merging**